### PR TITLE
Feature/styling issues

### PR DESCRIPTION
**Pull request by dcochico**
## What I did:
- Fix shifting issues when clicking delete buttons
- Fix shifting issues when toggling views
- Fix shifting issues when disabling cells
- Hide scrollbar in sidebar
- Add scrollbar to list and make invisible
## Did this break anything?
- [x] No
- [ ] Yes

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [x] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

## Checklist:
- [x] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [x] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

### CODE REVIEW

File: src/components/Grid/Grid.scss [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  display: grid;
+  grid-template-columns: repeat(10, 10%);
+  background-color: #beab95;
+  height: calc(100vh - 190px);
+  width: calc(100vh - 190px);
+  border-radius: 15px;
+  flex-shrink: 0;
+  justify-items: center;
+  align-items: center;
+  grid-auto-rows: minmax(0, 1fr);
```
Comment (mbenfowler) [inline src/components/Grid/Grid.scss:11]: what's this line do?

File: src/components/List/List.scss [hunk 2]
```diff
@@ -40,6 +40,11 @@
   width: calc(100vh - 190px);
   border-radius: 15px;
   flex-shrink: 0;
+  overflow-y: scroll;
+}
+
+.list-container::-webkit-scrollbar {
+  display: none;
```
Comment (mbenfowler) [inline src/components/List/List.scss:47]: what's this doing?

**Review by mbenfowler — APPROVED** [review]
just a couple clarifying q's - looks great!

File: src/components/Grid/Grid.scss (hunk 1 above)
Comment (dcochico) [reply src/components/Grid/Grid.scss:11]: line 11 keeps the gap between rows consistent regardless of enabled/disabled cells. this fixed the cell shifting bug

File: src/components/List/List.scss (hunk 2 above)
Comment (dcochico) [reply src/components/List/List.scss:47]: makes scrollbar invisible

**Pull request merged**